<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogUi</class>
 <widget class="QDialog" name="LogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="logTypeLA">
       <property name="text">
        <string>Log &amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>logTypeCO</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="logTypeCO"/>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>161</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextBrowser" name="logTB"/>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>201</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextErrorPB">
       <property name="toolTip">
        <string>Jump to the next error message.</string>
       </property>
       <property name="text">
        <string>Next &amp;Error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextWarningPB">
       <property name="toolTip">
        <string>Jump to the next warning message.</string>
       </property>
       <property name="text">
        <string>Next &amp;Warning</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="findLA">
       <property name="text">
        <string>&amp;Find:</string>
       </property>
       <property name="buddy">
        <cstring>findLE</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="findLE">
       <property name="toolTip">
        <string>Hit Enter to search, or click Go!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findPB">
       <property name="text">
        <string>&amp;Go!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="openDirPB">
     <property name="text">
      <string>&amp;Open Containing Directory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>111</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="updatePB">
     <property name="toolTip">
      <string>Update the display</string>
     </property>
     <property name="text">
      <string>&amp;Update</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>logTB</tabstop>
  <tabstop>openDirPB</tabstop>
  <tabstop>updatePB</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
